### PR TITLE
Reader: Suppress features for Laughing Squid

### PR DESCRIPTION
**Pull request by blowery**
LS uses featured images in a weird way. Suppress them and rely on picking things up from content instead.

### CODE REVIEW

**Comment by blowery** [comment]
cc @fraying

**Comment by fraying** [comment]
YES

File: client/lib/post-normalizer/rule-safe-image-properties.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ import { makeImageURLSafe } from './utils';
 export default function safeImagePropertiesForWidth( maxWidth ) {
 	return function safeImageProperties( post ) {
 		makeImageURLSafe( post.author, 'avatar_URL', maxWidth );
+		if ( post.feed_ID === 12098 || +post.site_ID === 1065188 ) {
```
Comment (bluefuton) [inline client/lib/post-normalizer/rule-safe-image-properties.js]: Numbers! So magic.

Can we move these numbers out to constants and have a comment explaining why we're doing this in this particular case?

Comment (blowery) [inline client/lib/post-normalizer/rule-safe-image-properties.js]: this is more for testing, i'm not planning on actually landing this. If we want to go with it, it'll be an API change.

**Comment by samouri** [comment]
I'm a little confused about this.  @blowery can you go into more detail about what LS is doing with the featured_image that is weird? Can we come up with a more generalized solution? It is going to be time consuming (and less effective) to manage site-fixes one by one.

Something else I would be afraid of, is fixing all of the sites/feeds that the reader team cares about which would give us a better impression of reader than the avg user.

If we definitely cannot generalize, lets make it easier to add/remove funky sites.

For example, something like:

```javascript
// some jetpack sites do X with their featured_images so we discount them
const brokenFeaturedImages = [  /* laughing squid */ 12098, 1065188, ];
if ( contains( brokenFeaturedImages, post.site_ID) || contains( brokenFeaturedImages, post.feed_ID) ) {
  post.featured_image = '';
}

```

**Comment by blowery** [comment]
@samouri The general problem is that how post thumbnails are used by WordPress sites varies. The usage is controlled by the currently active theme and it generally falls into one of three camps:

1. A featured image is a part of the body of the post. It may or may not appear at the top of the post. In this case, we currently suppress the featured image from the top of the post to prevent it appearing twice
2. A featured image is not part of the body, but it does appear on the site's single post page, usually at the top, sometimes as a background header. 
3. A featured image is not part of the body and does not appear on the single post page. This usage is somewhat more rare (I think), and the feature is usually used as a thumbnail in a listing page, like the main loop or the archives. 

Laughing Squid falls into 3, where the feature is present, but doesn't appear to be used, other than for publicize. 

We have no way to tell which of these three modes the currently active theme uses. We could build a map by hand, but that's quite fragile.

**Comment by blowery** [comment]
r145551-wpcom

**Pull request closed without merging**